src/cata_tiles.cpp

```diff
@@ -3389,4 +3392,50 @@ std::vector<options_manager::id_and_option> cata_tiles::build_renderer_list()
     return renderer_names.empty() ? default_renderer_names : renderer_names;
 }
 
+bool cata_tiles::save_screenshot( const std::string &file_path ) const
+{
+    std::string current_file_path;
+    if( file_path.empty() ) {
+        // check the current '<world>/screenshots' directory exists
+        std::stringstream map_directory;
+        map_directory << g->get_world_base_save_path() << "/screenshots/";
+        assure_dir_exist( map_directory.str() );
+
+        // build file name: <map_dir>/screenshots/[<character_name>]_<date>.png
+        std::time_t time = std::chrono::system_clock::to_time_t( std::chrono::system_clock::now() );
```

Comment:
Why `chrono`? You don't seem to need precise time and use `<ctime>` anyways. Simple `time()` would do.